templates/Dataland/datasetlinks.html.twig

```diff
@@ -0,0 +1,14 @@
+<h2>Dataset Links:</h2>
+<div>
+    {% for link in dataset.datasetSubmission.datasetLinks %}
+        <p>
+            <b>Name:</b>{{ link.name }}<br>
+            <b>Description:</b>{{ link.description }}<br>
+            <b>Function:</b>{{ link.functionCode }}<br>
+            <b>URL:</b>
+            <a href="{{ link.url }}" target="_blank">

```

Comment:
Should we decompose the link and strip out any included protocol, then add the protocol + link?

Reply:
No, the protocol of the link may not be the same as the protocol. It's just a thing.